src/lmic/lmic.c

```diff
@@ -470,7 +470,7 @@ static void reportEventNoUpdate (ev_t ev) {
 
     // rxstart is critical timing; legacy onEvent handlers
     // don't comprehend this; so don't report.
-    if (pOnEvent != NULL && (evSet & (1u<<EV_RXSTART)) == 0)
+    if (pOnEvent != NULL && (evSet & (1ul<<EV_RXSTART)) == 0)
```

Comment:
Is there a particular reason why this one was not changed to `UINT32_C(1)` too?

Reply:
I was in a hurry to get onto #275.... just missed it.